examples/plugin/Hello_world/package.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
```

Comment:
Ignition doesn't use `package.xml` files, these are usually used by ROS projects. Mind removing it?

Reply:
Done